teuthology/report.py

```diff
@@ -367,6 +367,26 @@ def get_jobs(self, run_name, fields=None):
         response.raise_for_status()
         return response.json()
 
+    def get_job(self, run_name, job_id, fields=None):
```

Comment:
This seems so much like get_jobs that I wonder if it should be just a slight specialization of get_jobs(); maybe add a job_id=None parameter, None meaning "get them all"